graphics/opencolorio/Portfile

```diff
@@ -49,6 +49,8 @@ if {${os.platform} eq "darwin" && ${os.major} < 20} {
 
     patchfiles-append   patch-yaml-cpp-0.7.diff
 
+    patchfiles-append   patch-fix-endianness.diff
```

Comment:
where does this patch come from? If from upstream, please add a link; if not, please submit upstream to verify that they agree with the approach. 

Also should this patch always be applied or only on certain systems - I doubt this port is broken across the board.

Reply:
@reneeotten Upstream code does nonsense, it is simply wrong (which they acknowledge, see TODO there). But notice it is for a **legacy** version of `colorio`, therefore I do not submit anything to upstream.